Apps/phironnet/ironnet_connector.py

```diff
@@ -319,6 +325,51 @@ def _handle_irondefense_report_observed_bad_activity(self, param):
             self.debug_print("Reporting bad activity to IronDefense failed. Error: {}".format(action_result.get_message()))
             return action_result.set_status(phantom.APP_ERROR, "Reporting bad activity to IronDefense failed. Error: {}".format(action_result.get_message()))
 
+    def _handle_irondefense_get_alert_notifications(self):
+            self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+            # Add an action result object to self (BaseConnector) to represent the action for this param
+            action_result = self.add_action_result(ActionResult(dict()))
+
+            request = {
+                'limit': self._alert_limit
+            }
+            alert_cats = self._alert_categories
+            alert_subcats = self._alert_subcategories
+            alert_sev_lower = self._alert_severity_lower
+            alert_sev_upper = self._alert_severity_upper
+
+            # make rest call
+            ret_val, response = self._make_post('/GetAlertNotifications', action_result, data=request, headers=None)
+            if phantom.is_success(ret_val):
+                self.save_progress("Fetching alert notifications was successful")
+                # Filter the response and add into the data section
+                for alert_notification in response['alert_notifications']:
+                    if alert_notification['alert']:
+                        alert = alert_notification['alert']
+                        if alert['category'] not in alert_cats and alert['sub_category'] not in alert_subcats:
+                            if alert_sev_lower <= int(alert['severity']) <= alert_sev_upper:
+                                contnr = {
+                                    'name': str(alert["id"]) + ' Alert Notification',
```

Comment:
This should be be changed to
```suggestion
                                    'name': '{} Alert Notification'.format(alert_notification['alert_action']),
```
so it is consistent with Demisto.